src/notifications/ManagerLinux.cpp

```diff
@@ -21,6 +26,18 @@ NotificationsManager::NotificationsManager(QObject *parent)
          "org.freedesktop.Notifications",
          QDBusConnection::sessionBus(),
          this)
+  , hasMarkup_{std::invoke([this]() -> bool {
+          for (auto x : dbus.call("GetCapabilities").arguments())
+                  if (x.toStringList().contains("body-markup"))
+                          return true;
+          return false;
+  })}
+  , hasImages_{std::invoke([this]() -> bool {
+          for (auto x : dbus.call("GetCapabilities").arguments())
+                  if (x.toStringList().contains("body-images"))
+                          return true;
```

Comment:
Don't call this in the constructor. It may throw, call it in the body. Also please don't block initialization on this, but make it an async call.